src/relay.js

```diff
@@ -158,7 +158,7 @@ export function sequelizeConnection({
    * @return {String}          The Base64 encoded cursor string
    */
   let toCursor = function (item, index) {
-    let id = item.get(item.Model.primaryKeyAttribute);
+    let id = item.get(item.constructor.primaryKeyAttribute);
```

Comment:
breaks v3 support, we need both

Reply:
should be fixed, I've never touched v3 :)